hw_config.h

```diff
@@ -1440,6 +1440,51 @@
 # define SERIAL_BREAK_DETECT_DISABLED   1
 # undef USE_VBUS_PULL_DOWN
 
+/****************************************************************************
+ * TARGET_HW_PIXHAWK5X
+ ****************************************************************************/
+#elif  defined(TARGET_HW_PIXHAWK5X)
+
+# define APP_LOAD_ADDRESS               0x08008000
+# define BOOTLOADER_DELAY               5000
+# define INTERFACE_USB                  1
+# define INTERFACE_USART                1
+# define USBDEVICESTRING                "Pixhawk5X.x"
```

Comment:
Changing the VID/PID has ripple effects on the the Firmware and QGC. On the Firmware the board will need it's own board support. If the USBDEVICESTRING does not contain PX4 BL it will have issues. 
See  https://github.com/PX4/PX4-Bootloader/blob/d0c2b65f4feba2c87ad084b0d03084324d9409c7/hw_config.h#L11-L18

Reply:
@davids5 
Thanks for the reminder. I have add the prefix "PX4 BL"